bindata/network/multus/multus.yaml

```diff
@@ -299,6 +313,9 @@ spec:
             items:
             - key: daemon-config.json
               path: daemon-config.json
+        - name: host-run-multus-certs
+          hostPath:
+            path: /run/multus_certs
```

Comment:
This will be removed if a node reboots right?
I don't think we want to wipe the certs if they are still valid.

Reply:
Great observation. Turns out that we'll need a modification to ensure that we re-use it on the Multus end instead of re-request. This will certainly be an improvement, but we'll address it as a follow-on item.